src/utils/pause/mapScopes/findGeneratedBindingFromPosition.js

```diff
@@ -341,22 +339,31 @@ function mappingContains(mapped, item) {
   );
 }
 
-async function getGeneratedLocationRange(
+function getGeneratedLocation(pos, generatedLocations) {
+  const { sourceId, line, column } = pos;
+  return generatedLocations[sourceId][line][column];
+}
+
+function getGeneratedLocationRange(
   pos: { start: Location, end: Location },
-  source: Source,
-  sourceMaps: any
+  generatedLocations: any
 ): Promise<{
   start: Location,
   end: Location
 } | null> {
-  const start = await sourceMaps.getGeneratedLocation(pos.start, source);
-  const end = await sourceMaps.getGeneratedLocation(pos.end, source);
+  // console.log(`> getting locations ${pos.start.line} ${pos.end.line}`);
+  const start = getGeneratedLocation(pos.start, generatedLocations);
+  const end = getGeneratedLocation(pos.end, generatedLocations);
```

Comment:
We may want to delay this, because this is the logic I'm already going to be changing to updating how the ranges work. Maybe it would make more sense for me to make that a batched lookup?

Reply:
that works!